spec/timezone.html

```diff
@@ -626,7 +626,7 @@ <h1>BuiltinTimeZoneGetInstantFor ( _timeZone_, _dateTime_, _disambiguation_ )</h
       <h1>GetPossibleInstantsFor ( _timeZone_, _dateTime_ )</h1>
       <emu-alg>
         1. Let _possibleInstants_ be ? Invoke(_timeZone_, *"getPossibleInstantsFor"*, « _dateTime_ »).
-        1. Let _list_ be ? CreateListFromArrayLike(_possibleInstants_, « Object »).
+        1. Let _list_ be ? IterableToList(_possibleInstants_).
```

Comment:
this is actually normative, because https://tc39.es/ecma262/#sec-createlistfromarraylike indeed does the validation on each item.

Reply:
I agree, this is normative, it's one of the normative changes that I included in my slides last month as part of the conditions for Stage 3. (Even without the validation, CreateListFromArrayLike does Get(_obj_, *"length"*).) I was looking through the commit messages to see if I had mentioned anywhere that it was editorial, but I don't think that's the case?

Reply:
Nah it’s not a contradiction or anything - but i didn’t realize it was intentional. Per above, i think iterable items should be validated per-item, not post-iteration.